src/runtime/types/index.ts

```diff
@@ -65,10 +73,14 @@ export interface MessagesLoader {
 }
 
 export interface ConfigureOptions {
-  fallbackLocale: string;
-  formats?: Partial<Formats>;
-  initialLocale?: string;
-  loadingDelay?: number;
-  warnOnMissingMessages?: boolean;
-  ignoreTag?: boolean;
+  fallbackLocale: string | null | undefined;
+  formats: Formats;
+  initialLocale: string | null;
```

Comment:
```suggestion
  initialLocale: string | undefined;
```

Reply:
5.
It's the same case of 3. I modified to `undefined` and leave test as it is with `as any`.